docs/examples/python/partials/misc/upgrade_environment.py

```diff
@@ -0,0 +1,46 @@
+import typing
```

Comment:
do we want this so exposed?

Reply:
the environment upgrade? yeah of course i think so it's like anything else on the environment. i added this as a helper for us right now and/or example for on demand, but can remove this file

Reply:
I feel like the one in [helpers/misc.py](https://github.com/aptible/cloud-api-clients/pull/36/files/750ea85b584b9cb211c64203389bd036d68b708f#diff-8523e4f1396ed839a3b0f9f9ffac9856fddb72334e19951a5e53ab4009d01e34) is sufficient for us and we should have a bigger discussion around anything else..